docs/run_test/platform.rst

```diff
@@ -109,6 +109,39 @@ The advantage is that it can run all test cases of Azure. The shortage
 is that the VM name is fixed, and it should be node-0, so each resource
 group can put only one VM.
 
+To use existing deployment, follow the steps below:
+
+1. Start a run with the variable values set to following in the runbook:
```

Comment:
Please make it's supported in Azure.yml, and use command line to control those variables. That's easier to use and explain.